Source/Fuse.Controls.CameraView/iOS/CameraViewImpl.h

```diff
@@ -29,6 +29,10 @@ namespace fcv {
 		CAMERA_FACING_BACK = 1,
 	};
 
+	enum CameraFocusPoint {
+		OK = 1,
+	};
```

Comment:
Again, the intention of this is kinda unclear... Also, why does this have a value of 1, as opposed to the Uno-definition?

Reply:
Yeah, its not necessary for the iOS and android implementations, so I'll remove em